aqua/Freecell/Portfile

```diff
@@ -14,19 +16,20 @@ description             classic solitaire game
 long_description        The imaginatively-titled “Freecell” is a Mac OS X \
                         version of the classic solitaire game.
 
-homepage                http://web.archive.org/web/20080622113332/http://randomoracle.com/freecell/
-master_sites            macports
-use_bzip2               yes
-
-checksums               md5 05c41aa87c3a6c390f83c12db7c4ff3d \
-                        sha1 2e609fe0548f44a316f77f15db39581aad7a587f \
-                        rmd160 0dbb3cbb23730654244091f4683c51a590a373ab
+checksums               md5 3171359a348ffa6ac46014a61235ef6b \
```

Comment:
There is no need to retain an md5 checksum any longer.

Reply:
Fixed